src/test/test_runner_host.py

```diff
@@ -304,6 +304,216 @@ def test_docker_basic_run(self):
         shutil.rmtree(repo.working_dir, ignore_errors=True)
 
 
+class TestHostPodmanRunner(PopperTest):
+    def setUp(self):
+        log.setLevel("CRITICAL")
+
+    @unittest.skipIf(os.environ.get("ENGINE", "docker") != "podman", "ENGINE != podman")
+    def test_stop_running_tasks(self):
+        with PodmanRunner() as pr:
+            cmd = ["podman", "run", "-d"]
+            _, _, c1 = HostRunner._exec_cmd(
+                cmd + ["debian:buster-slim", "sleep", "10000", "-q"], logging=False,
+            )
+            _, _, c2 = HostRunner._exec_cmd(
+                cmd + ["alpine:3.9", "sleep", "10000", "-q"], logging=False
+            )
+            c1 = c1.rstrip()
+            c2 = c2.rstrip()
+            pr._spawned_containers.add(c1)
+            pr._spawned_containers.add(c2)
+            pr.stop_running_tasks()
+            c1_status_cmd = [
```

Comment:
you could instead single an issue command that asks for the containers that are running and then assert that that is empty